src/stylesheet/sassHandler.ts

```diff
@@ -80,6 +80,7 @@ export async function renderModule(props: IRenderModuleProps): Promise<IStyleshe
     includePaths: [path.dirname(module.absPath)],
     outFile: module.absPath,
     sourceMapContents: requireSourceMap,
+    indentedSyntax: module.absPath.split('.').pop() == "sass",
```

Comment:
how about `module.extension === ".sass"` ?

Reply:
Yes, it's right!
I made changes according to the git flow.
just one commit in logs